src/design-system/summary-expander/summary-expander.component.tsx

```diff
@@ -28,6 +29,7 @@ export const SummaryExpander = ({
   disabled,
   testId,
   children,
+  hasBorder = true,
```

Comment:
As mentioned offline. I think we could try to follow the existing pattern with `variant: 'outline' | 'grey' | 'plain'` so you will have `outline` by default (to avoid modifying Lace) and `plain` to satisfy your borderless case :)

Reply:
Since `grey` has no usecase, I don't think we need to have three options and using boolean also follows the existing pattern. 🙂 . I'll suggest we leave this as boolean instead of a 3 options prop.

Reply:
Option 2 (if needed - a bit less consistent with current approach):
`plain` boolean which is by default `false` so the component renders a border but adding `<SummaryExpander plain />` satisfies the new requirement.

Reply:
Fixed on e6e6e27